Tools/_Starlight/partition_tests.py

```
@@ -328,12 +380,35 @@ def cmd_generate():
     class_counts = extract_classes(tests)
     print(f"Discovered {len(tests)} tests in {len(class_counts)} classes, distributing across {total} shards", file=sys.stderr)
 
+    timings = load_timings(timings_path)
```

Ruff (0.16.0)

[error] Undefined name `load_timings`

(F821)